doc/en/how-to/nose.rst

```diff
@@ -68,6 +68,34 @@ Unsupported idioms / known issues
   fundamentally incompatible with pytest because they don't support fixtures
   properly since collection and test execution are separated.
 
+Here is a table comparing the default supported naming conventions for both
+nose and pytest.
+
+========================== ======== ======
+Convention                  nose    pytest
+========================== ======== ======
+Ⓜ test*.py                  ✅
+Ⓜ test_*.py                 ✅       ✅
+Ⓜ \*_test.py                         ✅
+Ⓜ \*_tests.py
+Ⓒ \*\(unittest.TestCase\)   ✅       ✅
+ⓜ test_\*                   ✅       ✅
+Ⓒ Test\*                             ✅
+ⓜ test_\*                            ✅
```

Comment:
Perhaps I'm missing something but isn't this the same as two lines above?

Reply:
Well spotted! 🙇 
I looked at the source material as at this point it's a bit of a blur.
I'll need to fix the indentation somehow!
I think the original slide demonstrates it better:
<img width="912" alt="Screenshot 2021-04-06 at 13 51 56" src="https://user-images.githubusercontent.com/1423147/113713692-8947f580-96df-11eb-9a8a-0f6e2c8c0e87.png">

Reply:
I'm not too familiar with RST syntax, found this in the documentation [Escaping Mechanism](https://docutils.sourceforge.io/0.8/docs/ref/rst/restructuredtext.html#escaping-mechanism) but I don't think it would serve me here. If you have any suggestions that would be great!